docs/submissions.md

```diff
@@ -117,6 +117,10 @@ If a title was pre-screened at an event, regardless of location, the _**normal**
 For works such as one-shots, standalone movies, and anthologies, both the start and end date should be set to the same day. This gives it a '_Release Date_' label in the main entry.
 :::
 
+::: warning
+At times an anime will be announced using a >24 hour clock, such as saying the anime will start airing on July 3rd at 25:15. This typically will mean it is actually airing on July 4th at 1:15 AM, however this can also sometimes mean at July 3rd at 1:15 AM. Conversely they will announce an anime is airing on July 4th at 1:15 AM and that can sometimes mean it will air on July 3rd at 1:15 AM.
```

Comment:
```suggestion
Occasionally, an anime will be announced using a >24 hour clock, such as saying the anime will start airing on July 3rd at 25:15. This typically will mean it is actually airing on July 4th at 1:15 AM, however this can also sometimes mean at July 3rd at 1:15 AM. Conversely they will announce an anime is airing on July 4th at 1:15 AM and that can sometimes mean it will air on July 3rd at 1:15 AM.
```